db/migrate/20240817024535_create_relationships.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRelationships < ActiveRecord::Migration[7.1]
+  def change
+    create_enum :friendship_status, ['Friends', 'Blocked']
+    create_table :relationships do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :friend, null: false, foreign_key: { to_table: :users }
+      t.enum :friendship, enum_type: 'friendship_status', null: false
+      t.timestamps
```

Comment:
Why do we want timestamps?

Reply:
I've removed timestamps, doesn't really make sense for to know when they are friends